packages/airnode-examples/integrations/coinmarketcap/README.md

```diff
@@ -0,0 +1,15 @@
+# CoinMarketCap example integration
+
+This CoinMarketCap cryptocurrency price example builds upon the foundations of the CoinGecko example by incorporating
+authentication with an `apiKey` security scheme.
+
+In order to focus on authentication, many of the other aspects of this example are similar to the CoinGecko example
+including a single Ethereum price endpoint, the `_times` reserved parameter for handling floating point prices, and much
+of the `secrets.env` file.
+
+New to this example is a `config.json` security scheme that authenticates a request with CoinMarketCap via their
+preferred method: a request header named `X-CMC_PRO_API_KEY`. Your API key is provided as a value via interpolation of
```

Comment:
```suggestion
preferred method - a request header named `X-CMC_PRO_API_KEY`. Your API key is provided as a value via interpolation of
```

Reply:
Also, maybe mention how to obtain this api key (and that it's actually free)?

Reply:
I left the comma because grammatically it clarifies the preceding independent clause, but I did implement your other suggestion by adding that the user can get an API key with a free Basic CoinMarketCap account.